src/dom-manipulation.ts

```diff
@@ -122,3 +122,30 @@ export function getGameDomElements(round: number) {
     whiteIndicatorsContainer,
   };
 }
+
+/**
+ * @description Update the value and the color of the tooltip when we hover the verify button
+ * @param tooltip the DOM tooltip wich contains a <p> with a <span> inside
+ * @param roundLeft the number of rounds left in the game
+ */
+export function updateTooltip(tooltip: HTMLDivElement, roundLeft: number) {
+  const span = tooltip.querySelector('span');
+
+  if (!span) {
+    throw new Error('No span found in the tooltip');
+  }
+
+  span.innerHTML = `${roundLeft}`;
+
+  if (roundLeft <= 4) {
+    span.style.color = 'red';
```

Comment:
Maybe adapt it to be proportional to the number of turn max (if we begin at 7 turns max and it's already orange, it's strange

Reply:
Done